source/base/parameter_handler.cc

```diff
@@ -2379,6 +2402,37 @@ ParameterHandler::operator==(const ParameterHandler &prm2) const
 }
 
 
+std::set<std::string>
+ParameterHandler::get_entries_wrongly_not_set() const
+{
+  std::set<std::string> entries_wrongly_not_set;
+
+  for (auto it = entries_set_status.begin(); it != entries_set_status.end();
+       ++it)
+    if (it->second.first == true && it->second.second == false)
+      entries_wrongly_not_set.insert(it->first);
+
+  return entries_wrongly_not_set;
```

Comment:
This could be replaced by a iteration over the tree.

Reply:
Principally, I share the concern regarding the redundancy of data. However, implementing this iteration did not look so trivial to me when I had a look into the code. For me it is actually not worth implementing it but maybe it is much simpler than I thought. How many lines of code would that be? Question: Can you implement this without a copy and past of other code of the parameter handler and without investing a lot of effort into refactoring the ParameterHandler?